src/components/landing-page/call-to-action/CallToAction.jsx

```
@@ -1,22 +1,122 @@
+  whatOurUsersSay,
+} from "./Content";
+import "./responsive.css";
+import WhyChooseAurora from "../why-choose-aurora";
 
-const CallToActionPage = () => {
+function CallToActionPage() {
+  const [selectedLevel, setSelectedLevel] = useState("Beginner");
+  const { theme } = useTheme(); // Access the current theme
+
+  // Dark mode styles for cards
+  const darkModeCardStyles = theme === "dark" ? "bg-[#374151] text-white hover:bg-gray-700" : "";
```

ESLint

[error] 'darkModeCardStyles' is assigned a value but never used.

(no-unused-vars)